test/unit/lib/utils.js

```diff
@@ -0,0 +1,29 @@
+const chai = require('chai');
+const utils = require('../../../app/lib/utils');
+
+const expect = chai.expect;
+
+describe('utils', () => {
+  describe('deepClone', () => {
```

Comment:
Maybe, but the implementation can change and we have tests that cover the required functionality.

Why the comment about `poor man's clone`?